rest-api/dao/participant_summary_dao.py

```diff
@@ -379,6 +381,118 @@ def to_client_json(self, model):
 
     return result
 
+  def make_csv(self, results):
+    csv_data = StringIO.StringIO()
+    writer = unicode_csv.UnicodeWriter(csv_data)
+    headers = ps_full_data_headers
+    writer.writerow(headers)
+    for row in results.items:
+      line = self.to_client_csv(row)
+      writer.writerow(line)
+    return csv_data.getvalue()
+
+  def to_client_csv(self, row):
+    line = []
+    if (row.withdrawalStatus == WithdrawalStatus.NO_USE and
+      row.withdrawalTime < clock.CLOCK.now() - WITHDRAWN_PARTICIPANT_VISIBILITY_TIME):
+      row = {k: row.get(k) for k in WITHDRAWN_PARTICIPANT_FIELDS}
+
+    line.append(to_client_participant_id(row.participantId))
+    line.append(to_client_biobank_id(row.biobankId))
+    line.append(row.lastName)
+    line.append(row.firstName)
+    line.append(row.dateOfBirth)
+    if row.languageId:
+      language = self.code_dao.get(row.languageId).value
+      line.append(language)
+    else:
+      line.append(UNSET)
+    line.append(row.enrollmentStatus)
+    if row.consentForStudyEnrollment == 'SUBMITTED':
+      line.append(1)
```

Comment:
HPRO only uses the binary 1 or 0. This is where I needed @jt2k advice. It may be better to just return the string values (I think we are in `to_client_json` and let client alter data as they see fit.

Reply:
@shyamnaru opinion?

Reply:
If we are in `to_client_json` returning string values is fine.

FYI: If you want to return 1 or 0, for this perticular case healthpro checks for `SUBMITTED` string and asigns a value of 1 and for all others it's 0. https://github.com/vanderbilt/pmi-drc-hpo/blob/develop/src/Pmi/WorkQueue/WorkQueue.php#L364

Reply:
returning string values iot be explicit